modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -70,8 +70,27 @@ platform:
 publish: External
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
+compute:
+  platform:
+    ovirt:
+      ..
+      sparse: false # add this line
+      format: raw   # add this line
+controlPlane:
```

Comment:
The same applies to the `controlPlane` section. Should this content be added to the existing `controlPlane` top level section towards the start of this example instead?

Reply:
@Darth-Mera ??

Reply:
I apologize if my comment in the resolved conversation above was not clear enough - no repeating sections of the same name should be here. 

The whole Example default `install-config.yaml` file should look like this (if you really want to include it in this section and not in the custom machine pools section): 

```
----
apiVersion: v1
baseDomain: example.com
compute:
- architecture: amd64
  hyperthreading: Enabled
  name: worker
  platform:
    ovirt:
      sparse:
      format: 
  replicas: 3
controlPlane:
  architecture: amd64
  hyperthreading: Enabled
  name: master
  platform:
    ovirt:
      sparse:
      format: 
  replicas: 3
metadata:
  creationTimestamp: null
  name: my-cluster
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  machineNetwork:
  - cidr: 10.0.0.0/16
ifndef::openshift-origin[]
  networkType: OpenShiftSDN
endif::openshift-origin[]
ifdef::openshift-origin[]
  networkType: OVNKubernetes
endif::openshift-origin[]
  serviceNetwork:
  - 172.30.0.0/16
platform:
  ovirt:
    api_vip: 10.46.8.230
    ingress_vip: 192.168.1.5
    ovirt_cluster_id: 68833f9f-e89c-4891-b768-e2ba0815b76b
    ovirt_storage_domain_id: ed7b0f4e-0e96-492a-8fff-279213ee1468
    ovirt_network_name: ovirtmgmt
    vnicProfileID: 3fa86930-0be5-4052-b667-b79f0a729692
publish: External
pullSecret: '{"auths": ...}'
sshKey: ssh-ed12345 AAAA...
```